backend/LexBoxApi/GraphQL/CustomTypes/ProjectMembersVisibilityMiddleware.cs

```diff
@@ -0,0 +1,24 @@
+using HotChocolate.Resolvers;
+using LexCore.Entities;
+using LexCore.ServiceInterfaces;
+
+namespace LexBoxApi.GraphQL.CustomTypes;
+
+public class ProjectMembersVisibilityMiddleware(FieldDelegate next)
+{
+    public async Task InvokeAsync(IMiddlewareContext context, IPermissionService permissionService)
+    {
+        await next(context);
+        if (context.Result is List<ProjectUsers>)
```

Comment:
can this be more permissive? right now it's a List, but if it gets changed I don't want this to break, can we use `IEnumerable<ProjectUsers>`?

Reply:
Yes, that'll work.